data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,743 @@
+         _([[Last night however, we sighted a new island. We saw smoke hovering over it. The black comes from the fiery mountain in the north of the island, the grey however comes from settlers.]])),
+      h=500,
+   },
+   {
+      title = _("Planning For The Future"),
+      body = jundlina(_("Jundlina concludes"),
+         -- TRANSLATORS: Jundlina - Diary
+         _([[At least the east side of the island is inhabited, that much is certain. We do not know if the people there are friends or foes. We do not even know what tribe they might be from.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Jundlina - Diary
+         _([[The only thing we know for sure is that they cannot be Atlanteans, for no ship of ours has sailed as far as we did for centuries. I can only hope they are friendly.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Jundlina - Diary
+         _([[We have not much water left; our food is running low; we lost all our tools and most of our building materials with the third ship. We can’t afford to sail any further, and we must attempt to settle on this island as close as possible to our current position.]]))
+         .. new_objectives(obj_find_port),
+      h=500,
+   },
+} -- end of initial messages.
+
+initial_messages1 = {
+   {
+      title = _("Jundlina Gives Orders"),
+      body = jundlina(_("Jundlina decides"),
+         -- TRANSLATORS: Jundlina
+         _([[As we founded a port here, we can now plan our settlement. We still have plenty of iron ore, so a toolsmithy and a smelting works will quickly provide us with good tools.]]))
+         .. new_objectives(obj_tools),
+      h=500,
+   },
+   {
+      title = _("Jundlina Gives Orders"),
+      body = jundlina(_("Jundlina demands"),
+         -- TRANSLATORS: Jundlina
+         _([[However, we need to take care of our building materials: The island is full of trees, we might not even need foresters. But we need stone, that much is certain!]]))
+         .. new_objectives(obj_basic_infrastructure),
+      h=500,
+   },
+   {
+      title = _("Jundlina Gives Orders"),
+      body = jundlina(_("Jundlina orders"),
+         -- TRANSLATORS: Jundlina
+         _([[Let’s also explore this island and expand our territory. But we need to be careful. We should build at least one tower so that we can see potential enemies before they see us.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Jundlina
+         _([[But we need to be cautious using the stones and other precious material. When we no longer need a building, we will dismantle it instead of burning it down.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Jundlina
+         _([[This will take more time, but we can reuse some of its materials. Also, I want a scout deployed and exploring at all times.]]))
+         .. new_objectives(obj_explore, obj_expand),
+      h=500,
+   },
+} -- end of initial messages.
+
+function spidercloth_1(leader)
```

Comment:
All these functions need a pair of push/pop_textdomain calls

Reply:
I was so happy it works I forgot to implement . ;-) will do it now